src/pages/api/save-file.js

```diff
@@ -35,6 +35,7 @@ export default async function handler(req, res) {
     let fileContent = await fs.readFile(`${req.body.path}/${fileName}.tsx`, {
       encoding: 'utf-8',
     })
+    fileContent = fileContent.replaceAll(`// 🚨 Your props contains invalid code\n`, '')
```

Comment:
lgtm. 
`I hope you have tested it thoroughly and no extra new lines are added.`

Reply:
I have tested, no extra lines were added, won't say thoroughly but let's see